sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,484 @@
+
+To use an API key as the `credential` parameter, pass the key as a string into an instance of [AzureKeyCredential][azure_sdk_python_azure_key_credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.vision.face import FaceClient
+
+endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+face_client = FaceClient(endpoint, credential)
+```
+
+#### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [azure-identity][azure_sdk_python_identity] library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain] name for your resource in order to use this type of authentication.
+
+To use the [DefaultAzureCredential][azure_sdk_python_default_azure_credential] type shown below, or other credential types provided with the Azure SDK, please install the `azure-identity` package:
+
+```
+pip install azure-identity
+```
+
+You will also need to [register a new AAD application and grant access][register_aad_app] to Face by assigning the `"Cognitive Services User"` role to your service principal.
+
+Once completed, set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```python
+"""DefaultAzureCredential will use the values from these environment
+variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+"""
+from azure.ai.vision.face import FaceClient
+from azure.identity import DefaultAzureCredential
+
+endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
+credential = DefaultAzureCredential()
+
+face_client = FaceClient(endpoint, credential)
+```
+
+## Key concepts
+
+### FaceClient
+
+`FaceClient` provides operations for:
+
+ - Face detection and analysis: Detect human faces in an image and return the rectangle coordinates of their locations,
+   and optionally with landmarks, and face-related attributes. This operation is required as a first step in all the
+   other face recognition scenarios.
+ - Face recognition: Confirm that a user is who they claim to be based on how closely their face data matches the target face.
+   It includes Face verification ("one-to-one" matching) and Face identification ("one-to-many" matching).
+ - Finding similar faces from a smaller set of faces that look similar to the target face.
+ - Grouping faces into several smaller groups based on similarity.
+
+### FaceAdministrationClient
+
+`FaceAdministrationClient` is provided to interact with the following data structures that hold data on faces and
+persons for Face recognition:
+
+ - PersonDirectory
+ - FaceList
+ - LargeFaceList
+ - PersonGroup
+ - LargePersonGroup
```

Comment:
There is no operation group named as up under `FaceAdministrationClient` for source code.

Reply:
Thank you for pointing this out. I did some updates at commit 0309e20e50e376e577972c0ca56e0a645a7ec42a. 
In fact, that's how we call these data objects in our official documentation, so I hesitated slightly to make this change. What're your thoughts about it?
https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/add-faces
![image](https://github.com/Azure/azure-sdk-for-python/assets/100277363/da37786b-b058-483f-b6a2-261912bc75ab)

Reply:
In READEME.md for package user, it shall shows the API name and usage scenario. For example, `FaceAdministrationClient ` has API `create_face_list`, then you can list it here and describes its hero usage. Currently, you only list `face_list` without any description which confuse users.

Reply:
Added more introduction about all objects, including `face_list`.